### DISCUSSION TITLE
Use Secrets for adminPassword

### DISCUSSION THREAD

**Discussion by jairogutierrezpayu** [arkmq-org/arkmq-org-broker-operator#459] [General]
Hi, i tried use ENC(xxxx) notation for _adminPassword_ but i hadn't expected result, the result was literal value as console password,

I can use secret or encrypted password in my CRD?

bests,

**Comment by gaohoward** [comment]
Hi I think you can try using ActiveMQArtemisSecurity to configure your admin user and password. It supports store password in secrets, and your passwords will be encrypted.

**Comment by brusdev** [comment]
For admin credentials you can create a Secret named `<cr-name>-credentials-secret` with the admin and cluster user credentials, i.e.:
```
oc create secret generic broker-credentials-secret --from-literal=AMQ_USER=admin --from-literal=AMQ_PASSWORD=password --from-literal=AMQ_CLUSTER_USER=clusterUser --from-literal=AMQ_CLUSTER_PASSWORD=clusterPassword
```
